src/MizarData/include/MizarData/MizarPairedData.h

```diff
@@ -118,6 +118,10 @@ class MizarPairedDataTmpl {
                                           action_on_callstack_events);
   }
 
+  [[nodiscard]] uint64_t CaptureDuration() const {
```

Comment:
Nit: Either add the unit to the function name (`CaptureDurationNs`) or (better) return a strong type like `std::chrono::nanoseconds`